rewrite-java/src/main/java/org/openrewrite/java/ChangeType.java

```diff
@@ -132,11 +154,10 @@ public boolean isAcceptable(SourceFile sourceFile, ExecutionContext ctx) {
         });
     }
 
-    private static class JavaChangeTypeVisitor extends JavaVisitor<ExecutionContext> {
+    private class JavaChangeTypeVisitor extends JavaVisitor<ExecutionContext> {
```

Comment:
Why the change to non static as opposed to expanding the constructor here? Now there's a bit of an awkward mix of passing through some outer class field values through the constructor, while directly accessing one field.

Reply:
I noticed other classes in this package (including `changePackage`) don't define this kind of inner visitors as static, but I agree it's not very neat. I'll revert the removal of static and add the field and adjust the constructor